rules/integrations/endpoint/elastic_endpoint_security_memory_signature_prevented.toml

```
@@ -0,0 +1,105 @@
+[metadata]
+creation_date = "2024/03/24"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.16.0"
+promotion = true
+updated_date = "2024/11/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time an Elastic Defend alert for memory signatures are received. Enabling this rule
+allows you to immediately begin investigating your Endpoint memory signature alerts. This rule identifies Elastic Defend
+memory signature preventions only, and does not include detection only alerts.
+"""
+enabled = true
+from = "now-10m"
+index = ["logs-endpoint.alerts-*"]
+interval = "5m"
+language = "kuery"
+license = "Elastic License v2"
+max_signals = 10000
+name = "Memory Signature - Prevented- Elastic Defend"
```

I would suggest something like Memory Threat since event.code:shellcode_thread is not always associated to a yara signature.

```suggestion
name = "Memory Threat - Prevented- Elastic Defend"
```